index.bs

```diff
@@ -485,6 +485,11 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 : <dfn>Biometric Authenticator</dfn>
 :: Any [=authenticator=] that implements [=biometric recognition=].
 
+: Bound credential
```

Comment:
nit: lacks \<dfn>  :)

Reply:
Yeah, I didn't want to add one since I never reference it as "bound credential". Does it seem alright to make this `<dfn lt="bound">Bound Credential</dfn>` and change the `<dfn>bound</dfn>` in the paragraph to `[=bound=]`?

Reply:
Opened #1048 with that change.

Reply:
PR #1048 updated and merged.